tests/integration-tests/src/VuFindTest/OaiPmh/HarvesterFactoryTest.php

```diff
@@ -161,7 +161,11 @@ public function testSSLVerifyPeer()
         $client = $this->getMockClient();
         $client->expects($this->once())
             ->method('setOptions')
-            ->with($this->equalTo(['sslverifypeer' => false, 'timeout' => 60]));
+            ->with($this->equalTo([
+                'sslverifypeer' => false, 
+                'timeout' => 60, 
+                'adapter' => 'Laminas\Http\Client\Adapter\Proxy'
```

Comment:
This style might be preferable:

```suggestion
                'adapter' => \Laminas\Http\Client\Adapter\Proxy::class
```